multi-node/aws/pkg/cluster/stack_template.go

```diff
@@ -589,6 +596,48 @@ func StackTemplateBody(defaultArtifactURL string) (string, error) {
 		"Description": "Specific availability zone (optional)",
 	}
 
+	par[parVpcCIDR] = map[string]interface{}{
```

Comment:
The cloud-config is rendered by CloudFormation using the parameters, it needs access to all the data. Remember we removed the golang templates and replaced it with `renderTemplate` to make the Launch Stack stuff work.